docs/en/sql-reference/functions/time-series-functions.md

```diff
@@ -0,0 +1,45 @@
+---
+slug: /en/sql-reference/functions/time-series-functions
+sidebar_position: 140
```

Comment:
140 determines the position in the sidebar (e.g. [here](https://clickhouse.com/docs/en/sql-reference/functions/time-window-functions)). To retain sorting, this page should go between "Strings" (170) and "Time Window" (175). Let's use 172.